domaintools_async/__init__.py

```diff
@@ -53,9 +59,8 @@ def __await__(self):
 
         return self
 
-    async def __aiter__(self):
```

Comment:
looks like this change was made alluded to in 3.6 and made permanent in 3.7: https://bugs.python.org/issue27243, as annoying as it is we'll need to maintain backward compatibility for the time being.

Reply:
Right, this *is* the backwards compatible approach, since this will work on 3.5.3, 3.6 and 3.7 (I just verified). Whereas, the current master is *not* compatible with 3.7.

Reply:
It looks like I was wrong and they actually made the change in 3.5.2, that does mean that backward compatibility would be lost for the 3.5.0 and 3.5.1, however as 3.5.6 is now out I think it's safe to simply update the minimal version to 3.5.2. Thanks for verifying!